tests/python_tests/test_sampling.py

```diff
@@ -148,6 +156,7 @@ class RandomSamplingTestStruct:
                                     '\n\nOpenVINO is a social networking tool. OpenVINO is a free virtualization service that works at scale. The tool provides the ability'
                                 ]
                             ]),
+                            marks=[pytest.mark.xfail(reason="assert ref_text == ov_text fails in CI.", strict=True, condition=sys.platform in ["darwin", "win32"])]),
```

Comment:
is it an issue because of different implementation of random in C++ on Ubuntu and Windows?

Reply:
Ok, as I see it's implemented in this PR https://github.com/openvinotoolkit/openvino.genai/pull/628